### PR TITLE
feat(ui): add components for the next service right section

### PR DESCRIPTION
**Pull request by ctjhoa**
# What does this PR do?

https://qovery.atlassian.net/browse/FRT-768

2023-09-26-170748_1654x414_scrot

2023-09-26-120750_1667x282_scrot

### CODE REVIEW

**Comment by acarranoqovery** [comment]
[Qovery Preview](https://www.qovery.com)
---

Qovery can create a Preview Environment for this PR.
To trigger its creation, please post a comment with one of the following command.

| Command | Blueprint environment |
|---|---|
| /qovery  preview 28c47145-c8e7-4b9d-8d9e-c65c95b48425 | [staging](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/28c47145-c8e7-4b9d-8d9e-c65c95b48425/services/general) |
| /qovery  preview {all\|UUID1,UUID2,...} | To preview multiple environments |

---

This comment has been generated from [Qovery](https://qovery.com) AI :robot:.
Below, a word from its wisdom :
> The backlog never ends, be at peace with it

File: libs/shared/ui/src/lib/components/section/section.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import { type VariantProps, cva } from 'class-variance-authority'
+import { type ComponentPropsWithoutRef, type ElementRef, forwardRef, useContext } from 'react'
+import { twMerge } from 'tailwind-merge'
+import { LevelContext } from '../heading/level-context'
+
+const sectionVariants = cva(['flex', 'flex-col'], {
```
Comment (RemiBonnet) [inline libs/shared/ui/src/lib/components/section/section.tsx:6]: I think it's not really necessary to have it, color and font size should be set on the children no? here it brings too much complexity

Comment (ctjhoa) [reply libs/shared/ui/src/lib/components/section/section.tsx]: https://github.com/Qovery/console/pull/855/commits/138fca8e3d98626ccb93f4d03efb8007c264e777

File: libs/shared/ui/src/lib/components/header/header.spec.tsx [hunk 2]
```diff
@@ -1,9 +1,14 @@
 import { render } from '__tests__/utils/setup-jest'
+import { Section } from '../section/section'
 import Header from './header'
 
 describe('Header', () => {
   it('should render successfully', () => {
-    const { baseElement } = render(<Header />)
+    const { baseElement } = render(
```
Comment (RemiBonnet) [inline libs/shared/ui/src/lib/components/header/header.spec.tsx:7]: You can update the `render` with `renderWithProviders`

File: libs/shared/ui/src/lib/components/header/header.tsx [hunk 3]
```diff
@@ -24,9 +25,7 @@ export function Header(props: HeaderProps) {
         <div className="flex flex-col gap-3">
           <div className="flex gap-2 items-center max-w-3xl">
             <Skeleton height={36} width={150} show={!title}>
-              <h1 className="font-bold text-neutral-400 text-3xl max-w-3xl truncate">
-                {title && <Truncate text={title} truncateLimit={50} />}
-              </h1>
+              <Heading className="max-w-3xl truncate">{title && <Truncate text={title} truncateLimit={50} />}</Heading>
```
Comment (RemiBonnet) [inline libs/shared/ui/src/lib/components/header/header.tsx:28]: The `truncate` className is really necessary here?

**Review by RemiBonnet — APPROVED** [review]

**Pull request merged**